Dockerfile

```diff
@@ -4,6 +4,11 @@ RUN apk update && apk add git
 
 RUN go get github.com/jstemmer/go-junit-report
 
+RUN go get gopkg.in/olivere/elastic.v5
+RUN go get github.com/mitchellh/mapstructure
+
 COPY . /go/src/github.com/cyverse-de/querydsl
 
-CMD go test -v github.com/cyverse-de/querydsl | tee /dev/stderr | go-junit-report
+WORKDIR /go/src/github.com/cyverse-de/querydsl
+
+CMD go test -v ./... | tee /dev/stderr | go-junit-report
```

Comment:
Just yet this isn't a meaningful change, but the hope is to add tests for the subpackages which otherwise wouldn't be tested.